.circleci/config.yml

```diff
@@ -101,6 +101,22 @@ jobs:
             export PATH="$GOBIN:$PATH"
             make test_sim_modules
 
+  test_sim_gaia_nondeterminism:
```

Comment:
I actually don't think we want this. It being tested in test_cover makes more sense imo. I think having a parameterized make file command also makes sense to do though.